### PR TITLE
1.0.0 doc changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
- Rewrote usage, readme, and setup
    - Added navigation links

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -3,58 +3,166 @@
...
+    <td><strong>IMPORTANT</strong></td>
+    <td>The <code>JSON</code> data to be rendered must have <strong><em>at least one</em></strong> of the following:
+        <ul>
+            <li><code>data</code> key</li>
+            <li><code>title</code> and <code>value</code> keys</li>
+        </ul>
+    </td>
+</table>
+
+Following are two acceptable JSON formats:
```
Comment (cs-yash-patodkar) [inline docs/usage.md]: Both title and data must be present

File: docs/usage.md [hunk 2]
```diff
@@ -3,58 +3,166 @@
...
+
+<table>
+    <thead>
+        <th>With <code>data</code> Key</th>
+        <th>Without <code>data</code> Key</th>
+    </thead>
+    <tr>
+        <td>
+<pre>"recordTilesCustomerData": {
```
Comment (cs-yash-patodkar) [inline docs/usage.md]: You have used "summary-tile" data, use "summary-card" data, it is just below, there is no significance of 
the key-"value" in this widget

Use this as example
"recordSummaryCard": {
      "data": {
        "MTTD": "1 min",
        "MTTR": "2 mins",
        "Node Status": "Active",
        "Active Incidents": 0
      },
      "title": "Australia"
    }

File: docs/setup.md [hunk 3]
```diff
@@ -3,22 +3,26 @@
...
+| Fields                                  | Description                                                                                                                         |
+|-----------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------|
+| Title                                   | Specify a title for the record summary card.                                                                                        |
+| Data Source                             | Select the module containing JSON data.                                                                                             |
+| Select Field                            | Select a field from the module that contains the data in JSON format.                                                               |
+| Filter  Condition                       | Filter records containing the JSON data. It is recommended to apply filters that return only 1 record i.e. by using a unique field. |
```
Comment (cs-yash-patodkar) [inline docs/setup.md:19]: Not recommended for the Record-Summary-Card, rather if there are multiple records there will be multiple cards

**Review by cs-yash-patodkar — APPROVED** [review]

**Review by cs-suyog-jinturkar — APPROVED** [review]
Yash approved the changes and will incorporate doc comments in next PR

**Pull request merged**